eng/Version.Details.xml

```diff
@@ -30,7 +30,7 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>52fab5d72c6271f56121695fe780f6893d3ed7bd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.CoreFxTesting" Version="1.0.0-beta.18612.9">
+    <Dependency Name="Microsoft.DotNet.CoreFxTesting" Version="1.0.0-beta.18613.3">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>52fab5d72c6271f56121695fe780f6893d3ed7bd</Sha>
```

Comment:
You might want to update the Sha

Reply:
I never update the Sha, DARC can do that later for me.